shell/clean.sh

```diff
@@ -19,9 +17,7 @@ for directory in packages/*; do
 
             # Remove node modules
             rm -fr node_modules;
-
-            # Remove jspm packages
-            rm -fr jspm_packages;
+            rm -f yarn.lock;
```

Comment:
`clean` no longer runs on `make install`, because doing so would wipe out Yarn's really good dependency resolution. So the meaning of `make clean` has changed to "I really want to nuke everything and re-generate all Yarn lock files".